src/api/status/public/js/build-log/terminal.js

```diff
@@ -20,9 +20,12 @@ const theme = {
   white: '#CCCCC6',
   yellow: '#FD971F',
 };
+
+const width = window.innerWidth;
+const height = window.innerHeight;
```

Comment:
This will work, but it duplicates what we already have with https://www.npmjs.com/package/xterm-addon-fit.  The right way to do this is to have the parent element that holds the terminal fill the available width and height, and then it will automatically use that.  Can you fix our CSS for the page so that it does this?

Reply:
Okay, I will try to fix CSS. Actually I have checked some information about why the xterm-addon-fit command invalid. One explanation is that the container does not open until the browser has checked the size, so the xterm-addon-fit command is invalid. That is why I check windows' size first.